### PR TITLE
Feature/show number failed qc samples

### PR DESCRIPTION
**Pull request by KochTobi**
This PR shows the number of failed qc samples in the application.

### CODE REVIEW

File: .qube.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ full_name: Jennifer Bödker
 email: jennifer.boedker@qbic.uni-tuebingen.de
 project_name: Sample Tracking Status Overview
 project_short_description: Gives a visual overview of sample statuses
-version: 0.2.1
+version: 0.3.0-SNAPSHOT
```
Comment (KochTobi) [inline .qube.yml:6]: @jenniferboedker why is this in here and not on development o.O

Comment (jenniferboedker) [reply .qube.yml:6]: I guess that was a mistake from my side

Comment (jenniferboedker) [reply .qube.yml:6]: Feel free to remove that here and add it to dev :)

**Review by jenniferboedker — APPROVED** [review]

**Comment by jenniferboedker** [comment]
solves #46

**Pull request merged**